File: Examples/iOS/AudioKitDemo/AudioKitDemo/Analysis/VocalInput.m
//
//  VocalInput.m
//  AudioKitDemo
//
//  Created by Aurelius Prochazka on 2/14/15.
//  Copyright (c) 2015 Aurelius Prochazka. All rights reserved.
//

#import "VocalInput.h"

@implementation VocalInput

- (instancetype)init
{
    self = [super init];
    if (self) {
        AKAudioInput *microphone = [[AKAudioInput alloc] init];
        [self connect:microphone];
        _auxilliaryOutput = [AKAudio globalParameter];
        [self assignOutput:_auxilliaryOutput to:microphone];
    }
    return self;
}

@end

File: Examples/iOS/Swift/AudioKitDemo/VocalInput.swift
//
//  VocalInput.swift
//  AudioKitDemo
//
//  Created by Nicholas Arner on 3/1/15.
//  Copyright (c) 2015 AudioKit. All rights reserved.
//

class VocalInput: AKInstrument{
    
    let auxilliaryOutput = AKAudio.globalParameter()
    
    override init() {
        super.init()
        
        let microphone = AKAudioInput()
        connect(microphone)
        assignOutput(auxilliaryOutput, to: microphone)
    }
}
